homeassistant/components/rfxtrx/__init__.py

```diff
@@ -191,7 +191,11 @@ async def async_setup_entry(hass, entry: config_entries.ConfigEntry):
 
     hass.data[DOMAIN][DATA_CLEANUP_CALLBACKS] = []
 
-    await async_setup_internal(hass, entry)
+    try:
+        await async_setup_internal(hass, entry)
+    except asyncio.TimeoutError:
+        # Library currently doesn't support reload
+        return False
```

Comment:
I would probably not even catch it here. Just let the loader handle it. But not sure what @balloob thinks.

Reply:
We should catch it, since the loader doesn't know what it means when it receives a timeout error, and so will print the stack trace. We should make sure we print a log error message with what's up.

Reply:
A log message makes sense